api/types/events/events.proto

```diff
@@ -1673,6 +1673,20 @@ message WindowsDesktopSessionEnd {
     string WindowsUser = 7 [ (gogoproto.jsontag) = "windows_user" ];
     // DesktopLabels are the labels on the desktop resource.
     map<string, string> DesktopLabels = 8 [ (gogoproto.jsontag) = "desktop_labels" ];
+    // StartTime is the timestamp at which the session began.
+    google.protobuf.Timestamp StartTime = 9 [
+        (gogoproto.stdtime) = true,
+        (gogoproto.nullable) = false,
+        (gogoproto.jsontag) = "session_start,omitempty"
+    ];
+    // EndTime is the timestamp at which the session ended.
+    google.protobuf.Timestamp EndTime = 10 [
```

Comment:
A bit odd that the field is called "end" time but the JSON tag is session "stop".

If this matches what we've done for SSH session end events, then I would leave as-is, otherwise, consider `EndTime`/`"session_end"` or `StopTime`/`"session_stop"`.

Reply:
Agree, but yeah it's just a verbatim copy of what we have for the SSH events.